### ISSUE TITLE
My Provider Bindings

### ISSUE THREAD

**Issue by masonry47** [flipboxfactory/saml-sp#157]
Hi @dsmrt!

We have changed our url and when creating a new service provider, when we save to get the new meta data, the bindings still show the previous URL.  

<img width="723" alt="Screen Shot 2022-02-22 at 4 15 49 PM" src="https://user-images.githubusercontent.com/48130527/155229436-85edfdca-698f-43b4-9d7c-5d19e82dd61e.png">

We are trying to figure out where they url comes from.  The entity ID does succesfully show the new domain, but the bindings do not.  We are on 2.73 with Craft 3.7.28.

Thanks in advance!

**Comment by dsmrt** [comment]
👋 @masonry47 

The provider is set to the site in that my provider view. Does the site url for that site match what’s showing there? 

Also, you hinted at this but i just want to be sure, did you resave that provider? The bindings are pulled from the metadata that is saved in the db. The metadata is regenerated on save. 

Hope this helps!

**Comment by masonry47** [comment]
Thanks @dsmrt.  The selected site in the "My Provider" view does not match what is showing up in the bindings.  It still pulls in the previous URL for the site.  The old url is still in the general config file as it is being used in a different environment.  But again the entity ID does show the correct URL, just the bindings urls are off.  Thanks again in advance!

**Comment by dsmrt** [comment]
And have you resaved the provider in since the url was changed?

**Comment by masonry47** [comment]
Can you clarify?  Once the URL was changed, we simply went to 

![Screen Shot 2022-02-23 at 4 48 00 PM](https://user-images.githubusercontent.com/48130527/155422464-af0bdc46-e922-471e-8add-1e46811ec2d0.png)

And started the process over

Thanks!

**Comment by masonry47** [comment]
Thanks so much.  This helped.  We had thought we had changed that to an environment variable which would have the correct domain.  But it was just showing the old domain.  Oddly enough we tried to change it to an environment variable and then tried to change it to the new url, and neither would take and the base url field was stuck with the old domain. We were able to change it directly in the database, which did successfully update the base url field.  So maybe a Craft issue there.  But regardless one we got that value updated, the bindings showed the correct url on a new save.  

Thanks again!
